src/app/models/book-model.ts

```diff
@@ -0,0 +1,28 @@
+import { bookType } from '../book-type.enum';
+import { BookBorrowData } from './book-borrow-data-model';
+
+export class Book {
+  public id: number;
+  public nameBook: string;
+  public bookCategory: bookType;
+  public author: string;
```

Comment:
You can even think about Author class. But this is just a bonus.

Reply:
oki